package.json

```diff
@@ -36,6 +29,8 @@
     "@babel/preset-react": "^7.10.4",
     "@testing-library/react": "^11.0.4",
     "babel-loader": "^8.1.0",
+    "csv-writer": "^1.6.0",
```

Comment:
I was able to make my CSV without this. No big deal just letting you know it's possible.